assets/javascripts/kitten/components/images/image-with-legend.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import { Text } from 'kitten/components/typography/text'
+
+export class ImageWithLegend extends Component {
```

Comment:
Je sais que la PR est en `work in progress` mais je pense que ce composant devrait s'appeler `ImageWithCaption` plutôt.